scripts/e2e-test.sh

```diff
@@ -78,3 +78,144 @@ helm template deploy/helm/eunomia-operator/ \
   --set eunomia.operator.image.tag=dev \
   --set eunomia.operator.namespace=$TEST_NAMESPACE | kubectl delete -f -
 
+## Below block is to ensure all example are working
+helm template deploy/helm/eunomia-operator/ \
+  --set eunomia.operator.image.tag=dev \
+  --set eunomia.operator.image.pullPolicy=Never | kubectl apply -f -
+
+timeout=300
```

Comment:
Please set timeout separately before each test of example, if possible, instead of one timeout for all tests of examples.

Reply:
Agreed it will save time if any test fails .
Done with changes .